### DISCUSSION TITLE
Time duration for few videos is wrong

### DISCUSSION THREAD

**Discussion by moray5554** [Cyfrin/security-and-auditing-full-course-s23#11] [Q&A]
![Screenshot from 2023-12-08 16-39-34](https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/138766838/211c96d7-70c3-4dc0-a82b-54bb13c3f615)
As u can see on my screenshot left table of content says the duration of the video 'Self destruct' is 12 minutes, but the actual duration (u can see it on video-player) is 3m. Same problem with 'Upgradeable smart contract' left table says it 6 min, but in reality it is 11m.

**Comment by PatrickAlphaC** [comment]
Thank you! We are fixing this on the next PR
